internal/signalfx-agent/pkg/monitors/nagios/nagios.go

```diff
@@ -68,6 +68,8 @@ var (
 // Configure and kick off internal metric collection
 func (m *Monitor) Configure(conf *Config) error {
 	m.logger = logrus.WithFields(logrus.Fields{"monitorType": monitorType, "monitorID": conf.MonitorID})
+	m.logger.Warn("[NOTICE] The nagios monitor is deprecated and will be removed in a future release.")
```

Comment:
I believe we use `[WARNING]` in other places. Maybe better to be consistent if I'm not missing other places where we do `[NOTICE] `

Reply:
it's already logging at WARN level and therefore will show as a warning. We use [NOTICE] for all deprecation notices so far. 

https://github.com/search?q=repo%3Asignalfx%2Fsplunk-otel-collector%20%5BNOTICE%5D&type=code

You are correct that we use [WARNING], but only in the concept of config converters where we use log.Println:

https://github.com/search?q=repo%3Asignalfx%2Fsplunk-otel-collector+%5BWARNING%5D&type=code